Remove log from gitignore

rmoradi88 opened the pull request:
While publishing to pb.dev noticed that change.log has been added to gitignore. So, should remove it to be able to publish on pub.dev

td-famedly reviewed (changes requested):
No I think we just accidentally added the log file to the previous pub versions. I would just remove it from newer versions using the suggested pubignore fix

rmoradi88 commented:
> No I think we just accidentally added the log file to the previous pub versions. I would just remove it from newer versions using the suggested pubignore fix

Okay, according to https://dart.dev/tools/pub/publishing#what-files-are-published I added pubignore and added all ignored file since if both  ```gitignore``` and ```pubignore``` found in the directory only ```pubignore``` will be count.

td-famedly commented:
> > No I think we just accidentally added the log file to the previous pub versions. I would just remove it from newer versions using the suggested pubignore fix
> 
> Okay, according to https://dart.dev/tools/pub/publishing#what-files-are-published I added pubignore and added all ignored file since if both `gitignore` and `pubignore` found in the directory only `pubignore` will be count.

LGTM
